apps/src/code-studio/signup.js

```diff
@@ -0,0 +1,145 @@
+import $ from 'jquery';
+
+window.SignupManager = function (options) {
```

Comment:
Since it's only used on a single page, not sure that it's worth putting in code-studio.js, however it's very easy to stick into window.dashboard, so I can do that.